src/elasticsearch/controllers/course-webhook.controller.ts

```
@@ -0,0 +1,305 @@
+  })
+  @ApiResponse({ 
+    status: 500, 
+    description: 'Internal server error' 
+  })
+  async updateLMSCourseProgress(@Body() progressData: CourseProgressDto): Promise<{
+    success: boolean;
+    message: string;
+    data?: any;
+  }> {
+    try {
+      this.logger.log(`[LMS SERVICE] Received course progress update for user ${progressData.userId}, course ${progressData.courseId}`);
```

ESLint

[error] Replace ``[LMS·SERVICE]·Received·course·progress·update·for·user·${progressData.userId},·course·${progressData.courseId}`` with `⏎········`[LMS·SERVICE]·Received·course·progress·update·for·user·${progressData.userId},·course·${progressData.courseId}`⏎······`

(prettier/prettier)